istio-telemetry/kiali/values.yaml

```diff
@@ -47,15 +47,34 @@ kiali:
     #     - kiali.local
 
   dashboard:
-    secretName: kiali # You must create a secret with this name - one is not provided out-of-box.
+    auth:
+      strategy: login # Can be anonymous, login, or openshift
```

Comment:
Can you add `ldap` support in the comments?

Reply:
@clyang82 thanks for the spot.